napari_console/qt_console.py

```diff
@@ -150,13 +150,24 @@ def __init__(self, viewer: 'napari.viewer.Viewer'):
 
     def _update_theme(self, event=None):
         """Update the napari GUI theme."""
-        from napari.utils.theme import get_theme, template
+        from napari.utils.theme import get_theme
+        from napari.qt import get_stylesheet
+
+        # qtconsole unfortunately won't inherit the parent stylesheet
+        # so it needs to be directly set
+        if hasattr(self.viewer.window.qt_viewer, 'raw_stylesheet'):
```

Comment:
if we're going to add back the `raw_stylesheet` attribute with a deprecation warning in https://github.com/napari/napari/pull/2263 (which I did), then this PR isn't necessary at all.  When napari-console accesses `raw_stylesheet = self.viewer.window.qt_viewer.raw_stylesheet`, it will emit a DeprecationWarning that the end-user won't see.

If you want to keep compatibility with older napari while also being future proof, then just don't "opt-in" to the new `get_stylesheet` functionality, but avoid accessing `qt_viewer.raw_stylesheet` (which was never necessary in the first place since it could have always just been accessed at `get_stylesheet()`):

```
        raw_stylesheet = get_stylesheet()
        theme = get_theme(self.viewer.theme)
        self.style_sheet = template(raw_stylesheet, **theme)
```

Reply:
ah ok nice, I will use this approach